modules/cert-manager-acme-dns01-ambient-aws.adoc

```diff
@@ -102,38 +69,6 @@ spec:
 $ oc create -f issuer.yaml
 ----
 
-. Create a certificate:
-
-.. Create a YAML file that defines the `Certificate` object:
-+
-.Example `certificate.yaml` file
-[source,yaml]
-----
-apiVersion: cert-manager.io/v1
-kind: Certificate
-metadata:
-  name: <tls_cert>              <1>
-  namespace: <issuer_namespace> <2>
-spec:
-  isCA: false
-  commonName: '<common_name>'    <3>
-  secretName: <tls-cert>        <4>
-  dnsNames:
-  - '<domain_name>'              <5>
-  issuerRef:
-    name: <letsencrypt_staging>    <6>
-    kind: Issuer
-----
-<1> Provide a name for the certificate.
-<2> Specify the namespace that you created for the issuer.
-<3> Replace `<common_name>` with your common name (CN).
-<4> Specify the name of the secret to create that will contain the certificate.
-<5> Replace `<domain_name>` with your domain name.
-<6> Specify the name of the issuer that you created.
-
-.. Create the `Certificate` object by running the following command:
-+
-[source,terminal]
-----
-$ oc create -f certificate.yaml
-----
\ No newline at end of file
+[role="_additional-resources"]
```

Comment:
I see that there are some issues with this approach.

Adding the "additional section" resource after every section is not recommended. Please see, https://github.com/redhat-documentation/modular-docs/pull/210/files#diff-cad4c6a21f946e3004b72015660faab070c8cd54ecfc6468d0a393c1b76083c9R75 

> Only use . formatting (.Additional resources) in a module...

 https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

The link in the "additional resources" section is redirecting to the AWS Route53 section for every platform. For example, if users are referring to the GCP related section, they will be redirected to the AWS Route 53 section, which is confusing.

The removed steps are not exactly same for all the platforms. For example, `Create a certificate:` step is different for all sections.

It will be helpful for the users if they click on the section and then follow the exact procedure. 

I think the current structure works fine as it is. If you have any doubts please let me know. We can connect and discuss.

Reply:
Let me check and add the correct references accordingly. Thank you @xenolinux.